hw/ip/edn/doc/checklist.md

```diff
@@ -49,15 +49,15 @@ RTL           | [FEATURE_COMPLETE][]    | Done        |
 RTL           | [AREA_SMOKE_CHECK][]    | Done        |
 RTL           | [PORT_FROZEN][]         | Done        |
 RTL           | [ARCHITECTURE_FROZEN][] | Done        |
-RTL           | [REVIEW_TODO][]         | Pending     |
+RTL           | [REVIEW_TODO][]         | Done        |
 RTL           | [STYLE_X][]             | Done        |
 Code Quality  | [LINT_PASS][]           | Done        |
 Code Quality  | [CDC_SETUP][]           | N/A         |
 Code Quality  | [FPGA_TIMING][]         | Done        |
 Code Quality  | [CDC_SYNCMACRO][]       | N/A         |
-Security      | [SEC_CM_IMPLEMENTED][]  | Not Started |
-Security      | [SEC_NON_RESET_FLOPS][] | Not Started |
-Security      | [SEC_SHADOW_REGS][]     | Not Started |
+Security      | [SEC_CM_IMPLEMENTED][]  | Deferred    |
+Security      | [SEC_NON_RESET_FLOPS][] | Deferred    |
+Security      | [SEC_SHADOW_REGS][]     | Deferred    |
```

Comment:
No objection to that, but I'm personally not happy with taking away meaning from the D2 status in this way. Either we should globally remove these items from the D2 checklist, or simply wait with signing off as D2 until they are done.

Reply:
please confirm @mateenfaisal, but I have guidance to move to D2 assuming that security checks will be done later. I tend to agree that deferring these items changes the D2 milestone, but makes tracking readiness for a silver drop easier. Perhaps this direction of D2 has not been globally shared.